Gopkg.lock

```diff
@@ -904,7 +904,7 @@
     "pkg/userdata/ubuntu",
   ]
   pruneopts = "NUT"
-  revision = "1dd6b128854077daac70fb1a1d1975f365501604"
+  revision = "0f96e356b1f0ea9f7866547c1b5f9820e905a9ae"
   version = "v1.14.3"
```

Comment:
Should we bump the MC dependency to v1.15.0/v1.15.1 in [`Gopkg.toml`](https://github.com/kubermatic/kubermatic/blob/e2ff54983224a4a9e6ba96ac05525fe28d04aa63/Gopkg.toml#L139)?